react/src/lib/components/DeckGLMap/layers/intersection/intersectionView.stories.jsx

```diff
@@ -0,0 +1,86 @@
+import React from "react";
```

Comment:
I think we should make this a tsx file. It will make it easier to adapt the examples in a typescript environment.